src/verilog/verilog_expr.h

```diff
@@ -42,7 +42,7 @@ extern inline hierarchical_identifier_exprt
 class function_call_exprt : public binary_exprt
 {
 public:
-  function_call_exprt() : binary_exprt(ID_function_call)
+  function_call_exprt() : binary_exprt(exprt(), ID_function_call, exprt())
```

Comment:
Same here: this constructor really shouldn't be used, I think we need one that actually takes these expressions.

Reply:
Also removed.